KIPs/kip-7-fungible_token.md

```diff
@@ -119,6 +120,79 @@ interface IKIP7 {
     /// @param recipient The owner will receive the tokens.
     /// @param amount The token amount will be transferred.
     function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+    
+    /// @notice Moves `amount` tokens from the caller's account to `recipient`.
+    /// @dev Throws if the message caller's balance does not have enough tokens to spend.
+    /// Throws if the contract is pausable and paused.	
+    /// Throws if `_to` is the zero address. 
+    /// Returns a boolean value indicating whether the operation succeeded.
```

Comment:
we can use `@return` tag.
Refer https://solidity.readthedocs.io/en/develop/natspec-format.html

Reply:
In `safeTransfer`  function declaration, there is no return. 
We need to remove this tag line or add a return format in the function declaration.

Reply:
I removed it. :)

Reply:
Doesn't it still exist?

Reply:
Fixed. :) :)